inspirehep/modules/workflows/templates/inspire_workflows/index.html

```diff
@@ -77,13 +74,34 @@ <h3 class="text-center header-text">
 
       <holding-pen-dashboard-item
         template="{{url_for('static', filename='js/inspire_workflows_ui/templates/dashboard_stat_block.html')}}"
-        primary_filter_key="source"
-        primary_filter_value="arXiv"
+        filter_string="source=arXiv"
         secondary_filter="status"
         section_title="arxiv harvests">
       </holding-pen-dashboard-item>
 
     </div>
+
+    <div class="col-md-3 col-sm-6">
+
+      <holding-pen-dashboard-item
+        template="{{url_for('static', filename='js/inspire_workflows_ui/templates/dashboard_stat_block.html')}}"
+        filter_string="source=Alpha experiment"
```

Comment:
Not really, this is one of the sources that are crawled by the crawler, I added it because otherwise I would not be able to see the submissions on the dashboard (I think @eamonnmag is planning on generating them on the fly soon).